resources/supervisord.conf

```diff
@@ -10,3 +10,6 @@ stdout_logfile=/dev/stdout
 stdout_logfile_maxbytes=0
 stderr_logfile=/dev/stderr
 stderr_logfile_maxbytes=0
+
+[include]
+files = /etc/supervisor/conf.d/*.conf
```

Comment:
On `alpine` the path `/etc/supervisor/conf.d/*.conf` doesn't exist by default. Not sure about `slim`.
So what happens if it doesn't exist? Does supervisor fail? And should we put this in the readme that you can add extra supervisor configuration files on this path?

Reply:
Good point, and I did consider that. Actually that only results in a WARN and when you do a COPY of a file to that path no errors are given, so I thought a mkdir -p would be superfluous. That would be addressed if I actually make a separate xld.conf, which I'm even more leaning towards now ;)

Reply:
Re: documentation on the fragment dir: that is a great point.

Reply:
Makes sense. Because a separate `xld.conf` would also get rid of the warning. You wanna make the change before merge?

Reply:
Docs add. I'll make the change.